system/Database/BaseConnection.php

```diff
@@ -1406,9 +1406,38 @@ public function listTables(bool $constrainByPrefix = false)
     /**
      * Determine if a particular table exists
      */
-    public function tableExists(string $tableName): bool
+    public function tableExists(string $tableName, bool $cached = true): bool
```

Comment:
You have changed the public method. So please add the changelog
like https://codeigniter4.github.io/CodeIgniter4/changelogs/v4.2.2.html#breaking